pkg/reportingplugins/mercury/types.go

```diff
@@ -41,3 +41,14 @@ type Transmitter interface {
 	// - FromAccount() should return CSA public key
 	ocrtypes.ContractTransmitter
 }
+
+type ChainReader interface {
+	// LatestBlocks returns an ordered list of the latest specified number of blocks
+	LatestBlocks(context.Context, int) ([]Block, error)
+}
+
+type Block struct {
```

Comment:
Oh should we consider calling these `Head`s, since they are not full blocks.
```suggestion
type Head struct {
```

Reply:
Was keeping it consistent with the naming in mercury_v1 package, but i agree as they are not full blocks. BlockDescriptor? BlockHeader?

Reply:
There is precedent for some version of `Header` from geth. Not sure about others.

Reply:
Changed to Head. Thanks!